app/pages/vaccinations/data/production/vaccinations-solutions.json

```diff
@@ -44,6 +44,29 @@
         ]
       }
     },
+    {
+      "id": "10052-002",
+      "name": "SystmOne GP",
+      "summary": [
+        "The SystmOne Point of Care Additional Service supports the roll out of COVID-19 and flu vaccinations. Quick, accurate recording of all data directly into SystmOne. Combines familiar tools, for example, templates and questionnaires.",
+        "Seamless access to extra functionality allows for a smooth and more comprehensive experience for users and patients."
+      ],
+      "supplier": {
+        "id": "10052",
+        "name": "TPP – The Phoenix Partnership (Leeds) Ltd"
+      },
+      "vaccinations": {
+        "title": "How this Catalogue Solution can help with coronavirus vaccinations",
+        "list": [
+          "This Catalogue Solution provides an Additional Service that has been assessed for  vaccination and adverse reaction recording, appointments management and cohort identification",
+          "Records structured COVID-19 vaccination and adverse reaction data and adds directly to Patient Record",
+          "Supports checking availability and booking of COVID-19 vaccination appointments and records when they’re completed",
+          "Allows bulk sending of vaccination appointment invites via SMS, email and letters",
+          "Supports the cancellation and rescheduling of COVID-19 vaccination appointments",
+          "The Additional Service has met the Vaccination and Adverse Reaction Recording, Appointments Management - COVID-19 Vaccinations and Cohort Identification Capabilities"
```

Comment:
In 'Appointments Management - COVID-19', it looks like a hyphen-minus rather than an EN dash '–' between 'Management' and 'COVID-19'.